packages/ethereum-contracts/test/foundry/utils/BatchLiquidator.t.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity 0.8.18;
+
+import "../FoundrySuperfluidTester.sol";
+import { BatchLiquidatorV2 } from "@superfluid-finance/ethereum-contracts/contracts/utils/BatchLiquidator.sol";
+
+contract BatchLiquidator is FoundrySuperfluidTester {
+    using CFAv1Library for CFAv1Library.InitData;
+
+    BatchLiquidatorV2 batchLiquidator;
+    address liquidator = address(0x1234);
+
+    int96 FLOW_RATE = 10000000000;
+    uint256 FOUR_HOURS = 4 * 60 * 60;
+
+    constructor () FoundrySuperfluidTester(5) { }
+
+    function setUp() public override {
+        super.setUp();
+        batchLiquidator = new BatchLiquidatorV2(address(sf.host), address(sf.cfa));
+    }
+
+
+    // Helpers
+    function _startStream(address sender, address receiver, int96 flowRate) internal {
+        vm.startPrank(sender);
+        sf.cfaLib.createFlow(receiver, superToken, flowRate);
+        vm.stopPrank();
+    }
+
+    function _transferAllToSink(address sender) internal {
+        vm.startPrank(sender);
+        superToken.transferAll(admin);
+        vm.stopPrank();
+    }
+
+    function _assertNoFlow(address sender, address receiver) internal {
+        (,int96 flow,,) = sf.cfaLib.cfa.getFlow(superToken, alice, bob);
+        assertEq(flow, 0);
+    }
+
+    function testSingleLiquidation() public {
+
+        _startStream(alice, bob, FLOW_RATE);
+        _transferAllToSink(alice);
+
+        vm.startPrank(liquidator);
+        uint256 balance = superToken.balanceOf(liquidator);
+        vm.warp(FOUR_HOURS); // jump 4 hours
```

Comment:
is 4 hours because of liquidation period?
if it is, getting the liquidationPeriod for the particular token from governance will make the tests more robust
`(uint256 liquidationPeriod,) = sf.governance.getPPPConfig(sf.host, superToken);` this can be set in constructor

Reply:
4 hours because i just wanted to test the balance transfer itself. I'm not worry about liquidations payouts (that is tests on protocol) could be less